integration_test/cases/type.exs

```diff
@@ -147,6 +150,18 @@ defmodule Ecto.Integration.TypeTest do
     assert TestRepo.get!(Post, post.id).meta == %{"world" => "hello"}
   end
 
+  @tag :embedded
+  test "embeds one" do
+    profile = %Profile{location: "City"}
+    user = TestRepo.insert!(%User{profile: profile})
+    assert TestRepo.get!(User, user.id).profile.location == profile.location
+
+    user = %User{}
+    user = put_in(user.profile, build_embedded(user, :profile, location: "New"))
```

Comment:
Yeah, we should consider maybe make `build_embedded/3` already embed it. But let's discuss such APIs later.